ray-operator/Makefile

```diff
@@ -59,14 +59,14 @@ fumpt: gofumpt
 	$(GOFUMPT) -l -w ../
 
 ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
-test: manifests generate fmt vet fumpt ## Run tests.
```

Comment:
em. Why do revert the change here?

Reply:
this is some extra checks which we already done in our github flow.

Reply:
`make gofumpt` so we suggest user to manually run this if there're CI errors? that sounds fine.

Reply:
yes